### ISSUE TITLE
Payload source code

### ISSUE THREAD

**Issue by Ristovski** [bkerler/mtkclient#18]
Is the source code for all the payloads available somewhere?

The dump payloads can be found at https://github.com/chaosmaster/bypass_payloads (I assume they are the same), but it would be nice if all the `.bin`s had corresponding source code.

**Comment by Ristovski** [comment]
Just noticed that some of those are from https://github.com/MTK-bypass/exploits_collection/tree/master/payloads, for which the source code is at https://github.com/chaosmaster/bypass_payloads. There are still some that do not have source code though.

**Comment by bkerler** [comment]
No, actually chaosmaster and I decided not yet to release the source code of the payloads. The source published there is an old base. We will release them as soon we feel it's the right time.
